src/test/java/org/entando/k8s/db/job/CreateMysqlSchemaTest.java

```diff
@@ -14,13 +14,31 @@
 import java.sql.SQLException;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Tags;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
 
+@TestMethodOrder(OrderAnnotation.class)
 @Tags(@Tag("integration"))
 class CreateMysqlSchemaTest {
 
+    @Order(1)
+    @Test
```

Comment:
Isn't this a @Before instead?

Reply:
We'd need to refactor some method for instance method to static.
But in particular the before was not easily applicable to the oracle tests, so I've generalised the approach this way.